app/services/prismic.js

```diff
@@ -43,7 +43,7 @@ export default Service.extend({
     const documents = await api.query(PrismicJS.Predicates.at('document.type', 'news_item'), {
       page,
       pageSize,
-      orderings: '[document.first_publication_date desc]'
+      orderings: '[my.news_item.date desc]'
```

Comment:
A quoi ça sert `my` ?
Je pense que `orderings: '[my.news_item.inputDate desc]'` refléterait mieux le besoin métier

Reply:
Alors le `my` cela vient de la doc Prismic : https://prismic.io/docs/javascript/query-the-api/order-your-results
Et je suis pas totalement d'accord, car à partir du moment où ça vient de `my.news_item` c'est que ça vient d'une contribution Prismic